lib/ledger_sync/resources/txn.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module LedgerSync
+  class Txn < LedgerSync::Resource
```

Comment:
\+ @mattgmarcus 

Are we ok with this name?  I know `transaction` is a reserved word (we stupidly named our base ledger entry `transaction` at Paid, and it caused nightmares).  But like with `LedgerClass`, do we want to make this `LedgerTransaction`?

Reply:
I'm fine with `LedgerTransaction`

Reply:
Sounds good to me